tests/Aspire.Hosting.Tests/Redis/AddRedisTests.cs

```diff
@@ -111,6 +128,40 @@ public async Task VerifyManifest()
         Assert.Equal(expectedManifest, manifest.ToString());
     }
 
+    [Fact]
+    public async Task VerifyWithPasswordManifest()
```

Comment:
Do we also need tests for when `builder.Configuration["Parameters:pass"] = password` is missing?